renderlib/AppScene.h

```diff
@@ -95,6 +100,15 @@ class Scene
 
   Lighting m_lighting;
 
+  // convenience functions
+  // For now, this must match the order in which the lights were added, in initLights
+  static constexpr int SphereLightIndex = 0;
+  Light& SphereLight() const { return *m_lighting.m_Lights[SphereLightIndex]; }
+  SceneLight* SceneSphereLight() const { return m_lighting.m_sceneLights[SphereLightIndex]; }
+  static constexpr int AreaLightIndex = 1;
+  Light& AreaLight() const { return *m_lighting.m_Lights[AreaLightIndex]; }
+  SceneLight* SceneAreaLight() const { return m_lighting.m_sceneLights[AreaLightIndex]; }
+
```

Comment:
yeah currently one could do that.  It could happen if data were saved out badly, or corrupted after being saved, but I think that's the only way.   I could nail things down even more, of course.

Reply:
Honestly the way the light sources are handled right now is still sloppy in part because SceneObject (and SceneLight) is a new thing to this codebase.  A future PR will need to clean up the creation, deletion and handling of SceneObjects overall.